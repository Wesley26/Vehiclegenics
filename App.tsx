import { FC } from 'react';
import { TailwindProvider } from 'tailwind-rn';
import utilities from './tailwind.json';
import AppLoading from 'expo-app-loading';
import {
  useFonts,
  WorkSans_700Bold,
} from '@expo-google-fonts/work-sans';

import Home from './src/components/appPages/Home';

const App:FC = () => {

  let [activeFonts] = useFonts({
    //Expand active fonts when needed
    WorkSans_700Bold,
  });

  if (!activeFonts) return <AppLoading />;

  return (
    <TailwindProvider utilities={utilities}>
      <Home />
    </TailwindProvider>
  );
};

export default App;
