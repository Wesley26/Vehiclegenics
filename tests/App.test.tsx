import React from "react";
import renderer from "react-test-renderer";
import App from '../App';

//required so tailwindcss + tailwind-rn play nice with test
jest.useFakeTimers();

describe (`describing App.tsx component`, () => {

    it(`render App.tsx test component`, () => {

        const view = renderer.create(<App />);
        expect(view).toMatchSnapshot();

    });

});