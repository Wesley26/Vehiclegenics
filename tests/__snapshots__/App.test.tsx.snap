// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`describing App.tsx component render App.tsx test component 1`] = `null`;
